src/mockServer.ts

```diff
@@ -43,7 +40,12 @@ export async function startMockServer(params: MockServerParams): Promise<mockttp
   return server;
 }
 
-async function ensureCert(certPath: string, keyPath: string): Promise<void> {
+export async function ensureCert(testCaCertPath: string): Promise<{ certPath: string; keyPath: string }> {
+  const certPath = testCaCertPath;
+  const keyPath = path.join(path.dirname(certPath), `${path.basename(certPath, ".pem")}.key`);
+
+  await fs.mkdir(path.dirname(certPath), { recursive: true });
```

Comment:
lgtm, but curious - do the projects need manually be instrumented to ignore these files?

Reply:
> lgtm, but curious - do the projects need manually be instrumented to ignore these files?

Yeah. That's also the reason why the paths are configured externally.